### PR TITLE
Redirect uri and cookie consent

### PR DESCRIPTION
**Pull request by Rakeshcv**

### CODE REVIEW

File: src/Volvox.Helios.Web/Controllers/AuthenticationController.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class AuthenticationController : Controller
         public async Task<IActionResult> SignIn() => View("SignIn", await HttpContext.GetExternalProviders());
 
         [HttpPost("~/signin")]
-        public async Task<IActionResult> SignIn([FromForm] string provider)
+        public async Task<IActionResult> SignIn([FromForm] string provider, string ReturnUrl)
```
Comment (cjbose) [inline src/Volvox.Helios.Web/Controllers/AuthenticationController.cs]: Typically parameters are camel cased, so this should be `string returnUrl`

File: src/Volvox.Helios.Web/Controllers/HomeController.cs [hunk 2]
```diff
@@ -27,5 +27,10 @@ public IActionResult Index([FromServices] IDiscordUserService userService)
             
             return View(guilds.FilterAdministrator());
         }
+
+         public IActionResult Privacy()
```
Comment (cjbose) [inline src/Volvox.Helios.Web/Controllers/HomeController.cs]: A personal preference of mine is to use expression-bodied members for methods whose only function is to return a value. It definitely works as is, but can also be done like this.

`public IActionResult Privacy() => View();`

File: src/Volvox.Helios.Web/Views/Shared/_CookieConsentPartial.cshtml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                    <a asp-area="" asp-controller="Home" asp-action="Privacy" class="btn btn-info navbar-btn">Learn More</a>
+                    <button type="button" class="btn btn-default navbar-btn" data-cookie-string="@cookieString">Accept</button>
+                </div>
+            </div>
+        </div>
+</nav>
+    <script>
+        (function () {
+            document.querySelector("#cookieConsent button[data-cookie-string]").addEventListener("click", function (el) {
+                console.log("script");
```
Comment (cjbose) [inline src/Volvox.Helios.Web/Views/Shared/_CookieConsentPartial.cshtml]: References to console.log(); should be removed when you aren't debugging an issue.

File: src/Volvox.Helios.Web/Controllers/AuthenticationController.cs (hunk 1 above)
Comment (Rakeshcv) [reply src/Volvox.Helios.Web/Controllers/AuthenticationController.cs]: Sure thing!

File: src/Volvox.Helios.Web/Controllers/HomeController.cs (hunk 2 above)
Comment (Rakeshcv) [reply src/Volvox.Helios.Web/Controllers/HomeController.cs]: Sure.

File: src/Volvox.Helios.Web/Views/Shared/_CookieConsentPartial.cshtml (hunk 3 above)
Comment (Rakeshcv) [reply src/Volvox.Helios.Web/Views/Shared/_CookieConsentPartial.cshtml]: Sure.

**Comment by BillChirico** [comment]
Let's keep pull requests to one issue per. Could you remove the GDPR stuff?

**Comment by Rakeshcv** [comment]
Am gonna create separate pull request.

**Pull request closed without merging**